paddle/operators/trieconcat_op.h

```diff
@@ -0,0 +1,248 @@
+/* Copyright (c) 2016 PaddlePaddle Authors. All Rights Reserve.
```

Comment:
This operator does not need header file at all. You can put all code into .cc file

Reply:
use a header file is easier for testing.  because this op is a little complex, it need to be tested carefully in CPP.